internal/windows/user32dll_windows.go

```diff
@@ -27,6 +27,7 @@ var (
 	enumWindows                   = user32.NewProc("EnumWindows")
 	getActiveWindow               = user32.NewProc("GetActiveWindow")
 	getClientRect                 = user32.NewProc("GetClientRect")
+	getDC                         = user32.NewProc("GetDC")
```

Comment:
This, and the one for `releaseDC` should have a public wrapper put around them and exposed and those wrappers should be used instead of direct usage in display_windows.go to fit in with the way all the rest work.

Reply:
I removed GetDC and ReleaseDC because they are no longer necessary. EnumDisplayMonitors provides the DC in the callback.